lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -147,8 +147,8 @@ Function Add-ConstructedState($desired_state) {
 # ------------------------------------------------------------------------------
 Function Remove-ConstructedState($initial_state) {
   Try {
-    Get-ADComputer $initial_state.name `
-    | Remove-ADObject `
+    Get-ADComputer -identity $initial_state.name -credential $credential |
```

Comment:
Should be @extra_args like on line 120

Reply:
I thought it was clearer to use the "-credential" parameter but if you think it is better "@extra_args" I change it. Thank you

Reply:
I think it is better since there may be other parameters such as the server one you want to also have for the fix